servers/rendering/renderer_rd/environment/sky.cpp

```diff
@@ -276,7 +276,7 @@ void SkyRD::ReflectionData::update_reflection_data(int p_size, int p_mipmaps, bo
 	int mipmaps = p_mipmaps;
 	uint32_t w = p_size, h = p_size;
 
-	EffectsRD *effects = RendererCompositorRD::singleton->get_effects();
+	EffectsRD *effects = static_cast<RendererCompositorRD *>(RendererCompositorRD::get_singleton())->get_effects();
```

Comment:
I'll clean this bit up in 4.1, there are only 2 or 3 effects left that need their own little class and `get_effects` is toast.

Reply:
(I might sneak that in this week if I can, it's just cleanup so shouldn't break anything)